### ISSUE TITLE
Harbor slow to retrieve Projects/Repositories

### ISSUE THREAD

**Issue by jonhatallasf** [goharbor/harbor#12266]
I'm currently on 1.4.0 Helm Chart using harbor 2.0.x

We are experiencing very long wait times for seeing what docker images are stored in harbor and also the list of projects that have been created.  By long wait time, we often wait around 20 seconds for the UI to update.

```
core-8695d94df9-9vtwz core [D] [transaction.go:62] | 200 |10.001544161s|   match| GET      /api/v2.0/projects/2   r:/api/v2.0/projects/:id([0-9]+)
core-8695d94df9-tqgtb core  [DEBUG] [/server/middleware/log/log.go:30]: attach request id 2a6bad905198802cc7689c2786ae4b36 to the logger for the request GET /api/v2.0/projects/2
core-8695d94df9-tqgtb core [DEBUG] [/server/middleware/security/session.go:47][requestID="2a6bad905198802cc7689c2786ae4b36"]: a session security context generated for request GET /api/v2.0/projects/2
core-8695d94df9-tqgtb core [D] [transaction.go:62] | 200 |10.002680945s|   match| GET      /api/v2.0/projects/2   r:/api/v2.0/projects/:id([0-9]+)
```

We have scaled most components.  Things on my to-do list to try - standalone postgres instance

If anyone has any thoughts or has seen this before, any help would be greatly appreciated.  Also there is a guide to making pieces of harbor more HA - but nothing addressing performance.

**Comment by ninjadq** [comment]
Could you please provide more context like your configurations?

**Comment by jonhatallasf** [comment]
We have about 3 projects. 1 main project is our docker registry consisting of Around 200 images, with each image having between 1 and 2000 tagged versions. Most fall in a range between those numbers. We must keep all images for posterity and audit purposes. 

Another project is for helm charts and another we use for internally developed base images. 

That is the main setup.

**Comment by jonhatallasf** [comment]
@ninjadq  I have identified the culprit.  These requests are being blocked by an HTTP call to ChartMuseum

```
 2020-07-08T18:07:55Z [ERROR] [/core/api/project.go:257]: populate project properties failed with : get chart count of project 2 failed: get content failed: send request GET /docker/index.yaml failed: Get http://harbor-alpha-harbor-chartmuseum/docker/index.yaml: net/http: request canceled (Client.Timeout exceeded while awaiting headers)
```

Here is the workflow:
```
User:
  goes here: `https://<harbor-url>/harbor/projects`

Harbor: 
  Fetches harbor projects
  Fetches information about each project
  Tries to make a call for a given project to `http://<chartmuseum-service>/docker/index.yaml`
     - This seems to be a blocking call as the url waits until this returns (can take upwards of 10s)
```

Here is another issue experience the same pain, we can probably mark that or this as duplicate.
https://github.com/goharbor/harbor/issues/12272
